ui/apps/everest/src/pages/database-form/database-form-body/steps/resources/resources-step.tsx

```diff
@@ -22,7 +22,7 @@ export const ResourcesStep = () => {
         pairProxiesWithNodes={mode !== 'edit'}
         disableDiskInput={mode === 'edit'}
         allowDiskInputUpdate={mode !== 'edit'}
-        showSharding={dbType === DbType.Mongo && mode !== 'edit'}
+        showSharding={dbType === DbType.Mongo}
```

Comment:
sharding should not be available when editing a cluster. Any reason why we are removing this condition?

Reply:
@fabio-silva yes, new requirenments (https://perconadev.atlassian.net/browse/EVEREST-1306)
![image](https://github.com/user-attachments/assets/e2707621-6139-4706-bd98-6dded6567d5f)

we will not be able to enable/disable it, so toggle will be disabled but we should be able to change number of shards and number of config servers

Reply:
also in restoreToNewDb mode